Documentation/guides/messages/adb0000.md

```diff
@@ -1,20 +1,24 @@
-# Compiler Error/Warning ADB0000
+---
+title: Compiler error/warning ADB0000
+ms.date: 07/04/2018
+---
+# Compiler error/warning ADB0000
```

Comment:
I don't understand how/why this line changed.  Byte-order mark?

Reply:
There were a few cases of byte-order marks that changed. In this particular case though, the changes was just from lower-casing "Error" and "Warning" to follow [sentence-style capitalization](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/contributing-guidelines/template.md#headings). I debated on this change because the C# compiler error reference pages and NuGet error reference pages do capitalize "Error" and "Warning" in the titles. I'll be sure to get Craig's input about which option to choose before this PR is finalized.

On a separate question, is it OK to drop the byte-order marks from files that have them? The contribution guidelines don't specifically mention file encoding or byte-order marks. I spot-checked a few files in the C# compiler error reference pages and NuGet error reference pages and didn't see byte-order marks in those few, so it seems maybe it's most common not to include a byte-order marks. For now, I'll tentatively continue to leave out the byte-order marks for the next draft.

Reply:
BOM is not required - I would remove them if you notice them